modules/store/testing/src/mock_store.ts

```diff
@@ -18,46 +18,43 @@ import { MOCK_SELECTORS } from './tokens';
 if (typeof afterEach === 'function') {
```

Comment:
Do we still need this (because the static selectors aren't static anymore)?

Reply:
We still need it, because we don't know what test framework is using `MockStore` - it might not have `afterEach` teardown callback  🤷‍♂️
I'd still keep it.

Reply:
Sorry, I highlighted the wrong line 😅
It was more the question if we still have to cleanup, since we don't using static selectors anymore.

Reply:
Ah, yes, we still need to cleanup the reducers, because we set values on them (we keep them in MockStore as well, but we do set values on Memoized selectors that live outside).
I added the test that would fail if those are not cleaned up.

Reply:
Thanks!